client/jetpack-cloud/controller.ts

```diff
@@ -165,15 +164,11 @@ function recordNoVisibleJetpackSitesPageView(
 
 /**
  * Show dedicated screen if user has no Jetpack site, or no visible Jetpack site
- *
- * @param {string} siteFragment Parsed site slug
+ * @param {string|undefined} siteFragment Parsed site slug
```

Comment:
I thought we don't need JSDoc when we have types. Maybe we can remove the types from these `@param` tags.

Reply:
I agree with you, but this is not about one or two functions. It's a large scale issue in Calypso TypeScript code that types are duplicate in TS and in JSDoc. Better solve it separately about some wider discussion about how to best combine TS and JSDoc.

Reply:
Of course, just wanted to point it out while we're here 👍